dizoo/mujoco/config/walker2d_gail_sac_config.py

```diff
@@ -0,0 +1,103 @@
+from easydict import EasyDict
```

Comment:
why there is another [walker2d_gail_ddpg_config.py](https://github.com/opendilab/DI-engine/blob/1be0b4287c941f1767384cada5d7caa1dcec9c23/dizoo/mujoco/config/walker2d_gail_ddpg_config.py) here

Reply:
This is because when we did the GAIL reward experiments last year we used ddpg in combination with gail, so I thought it was convenient to leave it since it is a well-tested configuration. We used sac for the documentation benchmark